docs/tutorials/data-modeling-lifecycle/part-2-analytic-solution.md

```diff
@@ -121,37 +121,58 @@ for the `geoLocation` of the `WindTurbine` and `WindFarm`.
-| Point       | longitude      | float       |
-|             |                |             |
-| Polygon     | point          | Point       |
+Olav copies over all the properties from the enterprise model to the solution model. He then removes
+the rows that are not needed for the new solution model. The `Properties` spreadsheet will now look as follows
+(only the most relevant columns shown here):
+
+| Class       | Property       | Value Type  | ... | Reference                                  |
+|-------------|----------------|-------------|-----|--------------------------------------------|
+| WindTurbine | name           | string      |     | power:GeneratingUnit(property=name)        |
+| WindTurbine | type           | string      |     | power:GeneratingUnit(property=type)        |
+| WindTurbine | activePower    | timeseries  |     | power:GeneratingUnit(property=activePower) |
+| WindTurbine | geoLocation    | Point       |     | power:GeneratingUnit(property=geoLocation) |
+| WindTurbine | manufacturer   | string      |     | power:WindTurbine(property=manufacturer)   |
+| WindTurbine | ratedPower     | float       |     | power:WindTurbine(property=ratedPower)     |
+| WindTurbine | hubHeight      | float       |     | power:WindTurbine(property=hubHeight)      |
+| WindTurbine | lifeExpectancy | integer     |     | power:WindTurbine(property=lifeExpectancy) |
+|             |                |             |     |                                            |
+| WindFarm    | name           | string      |     | power:EnergyArea(property=name)            |
+| WindFarm    | geoLocation    | Polygon     |     | power:EnergyArea(property=geoLocation)     |
+| WindFarm    | windTurbines   | WindTurbine |     | power:WindTurbine(property=windTurbines)   |
 
 In addition, Olav removes the parent classes `GeneratingUnit` and `EnergyArea` and instead moves the properties he
-needs over to `WindTurbine` and `WindFarm`. The reason is that the `GeneratingUnit` and `EnergyArea` are generic
+needs over to `WindTurbine` and `WindFarm`. He moves the properties by renaming `GeneratingUnit` to `WindTurbine` and
+`EnergyArea` to `WindFarm` in the `Class` column. The reason is that the `GeneratingUnit` and `EnergyArea` are generic
 concepts that would complicate the solution model as this forecast model is only for wind turbines and cannot be
 applied to other types of generating units.
 
+Furthermore, Olav copies over the `GeoLocation`, `Point`, and `Polygon` classes from the enterprise model to the solution
+model. In addition, he writes up `WindTurbine` and `WindFarm`. The `Classes` spreadsheet will now look as follows:
+
+| Class       | Parent Class | ... | Reference         |
+|-------------|--------------|-----|-------------------|
+| GeoLocation | GeoLocation  |     | power:GeoLocation |
+| Point       | GeoLocation  |     | power:Point       |
+| Polygon     |              |     | power:Polygon     |
```

Comment:
this is something that would/will possible be simplified. It will be only necessary to refer to classes/views used in the model, where all parent classes/views will be automatically pulled by tracing inheritance.